apps/console/src/features/organizations/pages/organizations.tsx

```diff
@@ -101,6 +101,8 @@ const OrganizationsPage: FunctionComponent<OrganizationsPageInterface> = (
     const [ organization, setOrganization ] = useState<OrganizationResponseInterface>(null);
     const [ after, setAfter ] = useState<string>("");
     const [ before, setBefore ] = useState<string>("");
+    const [ authorizedListBefore, setAuthorizedListBefore ] = useState<string>("");
+    const [ authorizedListAfter, setAuthorizedListAfter ] = useState<string>("");
```

Comment:
Also, can't we use a single variable and distinguish between before and after cursor using the active page number or any other state variable?

Reply:
Separate variables are required for this to fetch authorized organization list.